l10n_ro_account_edi_ubl/models/account_move.py

```diff
@@ -22,6 +22,33 @@ class AccountMove(models.Model):
         help="ID used to download the ZIP file from ANAF.",
         copy=False,
     )
+    l10n_ro_edi_blocking_level = fields.Selection(
```

Comment:
este necesar un script de migrare pentru adaugarea unui camp calculat si stocat in notele contabile

Reply:
plus test